src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -5,22 +5,20 @@
 * The new HTTP and network semantic conventions can be opted in to by setting
   the `OTEL_SEMCONV_STABILITY_OPT_IN` environment variable. This environment
   variable supports the following values:
-  * `http` - emit the new, frozen (proposed for stable) HTTP and networking attributes, and stop
-  emitting the old experimental HTTP and networking attributes that the
-  instrumentation emitted previously.
-  * `http/dup` - emit both the old and the frozen (proposed for stable) HTTP and networking
-  attributes, allowing for a more seamless transition.
+  * `http` - emit the new, frozen (proposed for stable) HTTP and networking
+  attributes, and stop emitting the old experimental HTTP and networking
+  attributes that the instrumentation emitted previously.
+  * `http/dup` - emit both the old and the frozen (proposed for stable) HTTP
+  and networking attributes, allowing for a more seamless transition.
   * The default behavior (in the absence of one of these values) is to continue
   emitting the same HTTP and network semantic conventions that were emitted in
   `1.5.0-beta.1`.
   * Note: this option will eventually be removed after the new HTTP and
   network semantic conventions are marked stable. At which time this
   instrumentation can receive a stable release, and the old HTTP and
-  network semantic conventions will no longer be supported. A stable release of
-  this instrumentation will come no sooner than six months from now. This is to
-  allow for a transition period for users to experiment with the new semantic
```

Comment:
Why remove this part?

Reply:
Mostly just a mistake, but just moved it to the top. I think it reads better up there.